### PR TITLE
Added dummy changes.

### PR DESCRIPTION
**Pull request by rthoomu1**
These are dummy changes. Please ignore.

### CODE REVIEW

**Comment by rthoomu1** [comment]
Agreed. we will not merge.

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ Makefile.in
 
 # http://www.gnu.org/software/autoconf
 
+*.html
```
Comment (rthoomu1) [inline .gitignore]: Why are we excluding .html files. Please remove this line from here.

**Review by rthoomu1 — COMMENTED** [review]
address my changes

**Review by VNS117 — COMMENTED** [review]
Hi i reviewed files

**Pull request merged**